exercises/alphametics/tests/alphametics.rs

```diff
@@ -0,0 +1,64 @@
+extern crate alphametics;
+use std::collections::HashMap;
+
+fn test_helper(puzzle: &str, solution:  &[(char, u8)])  {
```

Comment:
is there (should there) be a better name for `test_helper`?

Then again, I don't have a particularly better one. `assert_solution_matches`?